run.ai/README.md

```diff
@@ -0,0 +1,77 @@
+# NIMs on Run.ai
+
+Run.ai provides a fast and efficient platform for running AI workloads. It sits on top of a group of Kubernetes clusters and provides UI, GPU-aware scheduling, container orchestration, node pooling, organizational resource quota management, and more. It gives customers the tools to manage resources across multiple Kubernetes clusters and subdivide them across project and departments, and automates Kubernetes primitives with its own AI optimized resources.
+
```

Comment:
<! -- Suggested re-wording & additions: --> 

[Run.ai](https://www.run.ai) provides a platform for accelerating AI development delivering life cycle support spanning from concept to deployment of AI workloads. It layers on top of Kubernetes starting with a single cluster but extending to centralized multi-cluster management.  It provides UI, GPU-aware scheduling, container orchestration, node pooling, organizational resource quota management, and more. And it offers administrators, researchers, and developers tools to manage resources across multiple Kubernetes clusters and subdivide them across project and departments, and automates Kubernetes primitives with its own AI optimized resources.

## Deployment Locations

The Run:ai Control Plane is available as a [hosted service](https://docs.run.ai/v2.18/home/components/#runai-control-plane-on-the-cloud) or alternatively as a [self-hosted](https://docs.run.ai/v2.18/home/components/#self-hosted-control-plane) option (including in disconnected "air-gapped" environments).  In either case, the control plane can manage Run:ai "cluster engine" equipped clusters whether local or remotely cloud hosted. 

<! -- Where is the DGX Cloud Run:ai control plane hosted?  This should be referenced here -->

Reply:
Can you double check those links on item 5? @resker

Reply:
I think I got it figured out, it was the extra bracket and parentheses after that I was questioning.